pkg/resource/model_package/testdata/model_package/v1alpha1/mp_invalid_before_create.yaml

```diff
@@ -29,6 +29,7 @@ spec:
                 s3URI: s3://source-data-bucket-592697580195-us-west-2/sagemaker/batch/input-data
             contentType: text/csv
           transformOutput:
+            kmsKeyID: ""
```

Comment:
desired may or may not specify these values, lets add unit tests separately and leave them out of this PR

and since the create failed, `mp_invalid_create_attempted.yaml` will also not have this value, since its coming as response of describe call

we will have to test this out when we have a readone or update related test

same for other unit test changes in this PR